client/package.json

```diff
@@ -36,6 +36,7 @@
     "ember-cli-eslint": "3.0.0",
     "ember-cli-flash": "1.3.17",
     "ember-cli-focus-input": "1.0.0",
+    "ember-cli-google-analytics": "^1.5.0",
```

Comment:
By "traffic to the site" do you mean anonymous traffic to the landing pages? Given Intercom already handles detailed tracking of users once they sign in.